boards/arm/ubx_bmd340eval_nrf52840/doc/index.rst

```diff
@@ -0,0 +1,518 @@
++-------+----------------+
+| 10    | EXT_RESET      |
++-------+----------------+
+
+Debug IN (J26)
+
++-------+----------------+
+| PIN # | Signal Name    |
++=======+================+
+| 1     | BMD-340_VCC    |
++-------+----------------+
+| 2     | BMD-340_SWDIO  |
++-------+----------------+
+| 3     | GND            |
++-------+----------------+
+| 4     | BMD-340_SWDCLK |
++-------+----------------+
+| 5     | GND            |
++-------+----------------+
+| 6     | BMD-340_SWO    |
++-------+----------------+
+| 7     | N/C            |
++-------+----------------+
+| 8     | N/C            |
++-------+----------------+
+| 9     | GND            |
++-------+----------------+
+| 10    | BMD-340_RESET  |
++-------+----------------+
+
+Auxiliary (J9)
+
++-------+----------------+
+| PIN # | Signal Name    |
++=======+================+
+| 1     | P0.10 / NFC2   |
++-------+----------------+
+| 2     | P0.09 / NFC1   |
++-------+----------------+
+| 3     | P0.08          |
++-------+----------------+
+| 4     | P0.07          |
++-------+----------------+
+| 5     | P0.06          |
++-------+----------------+
+| 6     | P0.05 / AIN3   |
++-------+----------------+
+| 7     | P0.01 / XL2    |
++-------+----------------+
+| 8     | P0.00 / XL1    |
++-------+----------------+
+
+Auxiliary (J10)
+
++-------+-------------------+
+| PIN # | Signal Name       |
++=======+===================+
+| 1     | P0.11 / TRACED[2] |
++-------+-------------------+
+| 2     | P0.12 / TRACED[1] |
++-------+-------------------+
+| 3     | P0.13             |
++-------+-------------------+
+| 4     | P0.14             |
++-------+-------------------+
+| 5     | P0.15             |
++-------+-------------------+
+| 6     | P0.16             |
++-------+-------------------+
+| 7     | P0.17 / QSPI_CS   |
++-------+-------------------+
+| 8     | P0.18 / RESET     |
++-------+-------------------+
+| 9     | P0.19 / QSPI_CLK  |
++-------+-------------------+
+| 10    | P0.20 / QSPI_D0   |
++-------+-------------------+
+| 11    | P0.21 / QSPI_D1   |
++-------+-------------------+
+| 12    | P0.22 / QSPI_D2   |
++-------+-------------------+
+| 13    | P0.23 / QSPI_D3   |
++-------+-------------------+
+| 14    | P0.24             |
++-------+-------------------+
+| 15    | P0.25             |
++-------+-------------------+
+| 16    | P1.00 / TRACED[0] |
++-------+-------------------+
+| 17    | P1.09 / TRACED[3] |
++-------+-------------------+
+| 18    | No connection     |
++-------+-------------------+
+
+Arduino Headers
+---------------
+
+Power (J5)
+
++-------+--------------+-------------------------+
+| PIN # | Signal Name  | BMD-34x Functions       |
++=======+==============+=========================+
+| 1     | VSHLD        | N/A                     |
++-------+--------------+-------------------------+
+| 2     | VSHLD        | N/A                     |
++-------+--------------+-------------------------+
+| 3     | RESET        | P0.21 / RESET           |
```

Comment:
In nRF52840, the nRESET function is on P0.18.

Reply:
Fixed - thanks for the catch!